todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/effecthandlers/TaskDetailEffectHandlers.java

```diff
@@ -53,7 +53,7 @@
   public static ObservableTransformer<TaskDetailEffect, TaskDetailEvent> createEffectHandlers(
       TaskDetailViewActions view, Context context, Action dismiss, Consumer<Task> launchEditor) {
 
-    TasksRemoteDataSource remoteSource = TasksRemoteDataSource.getInstance();
+    TasksRemoteDataSource remoteSource = TasksRemoteDataSource.INSTANCE;
```

Comment:
I'm probably missing something, but it doesn't look like `TasksRemoteDataSource` has an `INSTANCE` field